src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -153,6 +177,24 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         ColorSchemeComboBox().SelectedItem(scheme);
     }
 
+    void ColorSchemes::Rename_Click(IInspectable const& sender, RoutedEventArgs const& /*e*/)
+    {
+        const auto schemeName{ CurrentColorScheme().Name() };
+        const hstring title{ fmt::format(std::wstring_view{ RS_(L"ColorScheme_RenameDialog/Title") }, schemeName) };
+
+        auto dialog{ FindName(L"RenameDialog").try_as<ContentDialog>() };
+        dialog.Title(winrt::box_value(title));
```

Comment:
this looks like an `x:Uid` exploded and left shrapnel everywhere. What happened?

Reply:
ah, it's because you wanted to format the color scheme name into it. Sure, fine.